src/common/formUtils.ts

```diff
@@ -47,37 +47,6 @@ const includesGenres = (
   return hasValidGenre;
 };
 
-const createAudioBuffer = async (value: File) => {
-  const audioContext = new (window.AudioContext || window.AudioContext)();
-  const arrayBuffer = await value.arrayBuffer();
-
-  return audioContext.decodeAudioData(arrayBuffer);
-};
-
-const createAsyncAudioTest = (
-  testFn: (audioBuffer: AudioBuffer) => boolean
-) => {
-  return async (value: File) => {
-    if (!value) return false;
-
-    const audioBuffer = await createAudioBuffer(value);
-
-    return testFn(audioBuffer);
-  };
-};
-
-const isFileSizeValid = (value: File) => {
-  if (!value) return false;
-
-  const fileSizeInMB = value.size / (1024 * 1024);
-  const fileSizeInGB = value.size / (1024 * 1024 * 1024);
```

Comment:
Thinking about switching this and album max value to 1000 instead of 1024 factor, as files are typically in bytes and Megabytes/Gigabytes (MB/GB) not Memibytes/Gebibytes (MiB/GiB). Do others agree?